README.md

```diff
@@ -191,43 +191,31 @@ Execute the commands below to build from the source.
    ./gradlew clean test
    ```
 
-3. To run a group of tests
```

Comment:
Was excluding the guide for running tests on environments intentional?

Reply:
thought of removing the guide since it is not necessary for this package as there are no test groups now

Reply:
Yes, the guide for running tests on a different group should be removed. However, I'm asking about the guide for setting up the environments so users can test on live endpoints or mock servers (it's already available under tests/README.md ). Do we need to exclude it as well ?

Reply:
I have removed the details about test groups from the README.md file and included only instructions on setting up environmental variables to execute either live or mock tests.